R/enumpart.R

```diff
@@ -247,24 +264,29 @@ redist.calc.frontier.size <- function(ordered_path){
 #' @param ordered_path valid path to output the ordered adjacency map to
 #' @param out_path Valid path to output the enumerated districts
 #' @param ndists number of districts to enumerate
-#' @param all boolean. TRUE outputs all districts. FALSE samples n districts. 
-#' @param n integer. Number of districts to output if all is FALSE. Returns 
+#' @param all boolean. TRUE outputs all districts. FALSE samples n districts.
+#' @param n integer. Number of districts to output if all is FALSE. Returns
 #' districts selected from uniform random distribution.
+#' @param weight_path A path (not including ".dat") to a space-delimited file containing a vector of
+#' vertex weights, to be used along with \code{lower} and \code{upper}.
+#' @param lower A lower bound on each partition's total weight, implemented by rejection sampling.
+#' @param upper An upper bound on each partition's total weight.
 #' @param init Runs redist.init.enumpart. Defaults to false. Should be run on first use.
-#' @param read boolean. Defaults to TRUE. reads 
+#' @param read boolean. Defaults to TRUE. reads
 #' @param total_pop Integer Vector. Defaults to NULL. If supplied, computes the parity.
 #' @param adjlist Deprecated, use adj. zero indexed adjacency list
 #' @param ndist Deprecated, use ndists. number of districts to enumerate
 #' @param population Deprecated, use total_pop. Integer Vector. Defaults to NULL. If supplied, computes the parity.
 #'
 #' @return List with entries district_membership and parity.
-#' @export
+#' @concept enumerate
 #'
-redist.enumpart <- function(adj, unordered_path, ordered_path, 
-                             out_path, ndists = 2, all = TRUE, n = NULL, 
-                            init = FALSE, read = TRUE, 
+redist.enumpart <- function(adj, unordered_path, ordered_path,
```

Comment:
The function is still defined here, but is no longer in the namespace. Any thoughts on why?

Reply:
somehow the @export got removed during merge.  next commit will fix